zerver/lib/test_runner.py

```diff
@@ -225,13 +225,19 @@ def _replacement_destroy_test_db(self: DatabaseCreation,
                        % (self.connection.ops.quote_name(test_database_name),))
 DatabaseCreation._destroy_test_db = _replacement_destroy_test_db
 
-def destroy_test_databases(worker_id: int) -> None:
-    """Modified from the Django original to """
-    database_id = random_id_range_start + worker_id
+def destroy_test_databases(worker_id: Optional[int]=None) -> None:
     for alias in connections:
         connection = connections[alias]
         try:
-            connection.creation.destroy_test_db(number=database_id)
+            # We need this conditional otherwise the call to destroy_test_db gets
+            # confused on which db is being destroyed and tries to append
+            # `number` on to a clone.
+            if worker_id is not None:
+                """Modified from the Django original to """
```

Comment:
Hey you probably cut pasted this comment from the function beginning to this position. I think it belongs where it was before.